packages/datagrid/src/components/PlaygroundCellEditor/ApplyToIdenticalValues.scss

```
@@ -0,0 +1,14 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+.apply-to-identical-values {
+	min-width: 30rem;
+	box-shadow: tokens.$coral-elevation-shadow-neutral-m;
+	border-radius: tokens.$coral-radius-s;
+	padding: tokens.$coral-spacing-m tokens.$coral-spacing-s tokens.$coral-spacing-s;
+	background: tokens.$coral-color-neutral-background;
+
+	display: flex;
+	flex-direction: column;
+	row-gap: tokens.$coral-spacing-m;
+	z-index: 9999999;
```

```suggestion
	z-index: 1;
```
I jest. But also, do I?

---

Removed it, no longer needed since editor is now opened in a "popup" from ag grid's standpoint